core/libjsonnet.cpp

```diff
@@ -277,16 +284,14 @@ const char *jsonnet_version(void)
 
 JsonnetVm *jsonnet_make(void)
 {
-    TRY
-    return new JsonnetVm();
+    TRY return new JsonnetVm();
```

Comment:
I was able to tell clang-format that TRY ... CATCH is a block (as it is, no changes to the macro).